packages/cli-tool/src/helpers/file-editor.ts

```diff
@@ -0,0 +1,52 @@
+import fs from 'node:fs'
+
+export type lineFinderFuncType = (lines:string[])=>number;
```

Comment:
Would be good to reformat this file 🙂 
```suggestion
export type lineFinderFuncType = (lines: string[]) => number;
```

Reply:
I've rebased from #101 and reformat everything~ 66348b7 & f3a8ae5 ;-) 

thanks!